### PR TITLE
Feature: support relation CRUD operation

### PR DESCRIPTION
**Pull request by onlyjackfrost**
support relation CRUD operation

### CODE REVIEW

File: wren-ui/src/apollo/server/repositories/modelRepository.ts [hunk 1]
```diff
@@ -22,4 +24,8 @@ export class ModelRepository
   constructor(knexPg: Knex) {
     super({ knexPg, tableName: 'model' });
   }
+  findAllByIds = async (ids: number[]) => {
```
Comment (wwwy3y3) [inline wren-ui/src/apollo/server/repositories/modelRepository.ts:27]: add `public` to method.

File: wren-ui/src/apollo/server/repositories/relationshipRepository.ts [hunk 2]
```diff
@@ -178,4 +179,25 @@ export class RelationRepository
     );
     return result.map((r) => this.transformFromDBData(r)) as RelationInfo[];
   }
+
+  public async findDuplicateRelationBetweenModels(modelIds) {
```
Comment (wwwy3y3) [inline wren-ui/src/apollo/server/repositories/relationshipRepository.ts:183]: nit: maybe rename to `findExistedRelationshipBetweenModels`

**Review by wwwy3y3 — COMMENTED** [review]
Others LGTM

File: wren-ui/src/apollo/server/repositories/modelRepository.ts (hunk 1 above)
Comment (onlyjackfrost) [reply wren-ui/src/apollo/server/repositories/modelRepository.ts]: delivered in 511d5faf675f40bd516a00d5258178530bae5d5b

File: wren-ui/src/apollo/server/repositories/relationshipRepository.ts (hunk 2 above)
Comment (onlyjackfrost) [reply wren-ui/src/apollo/server/repositories/relationshipRepository.ts]: delivered in 511d5faf675f40bd516a00d5258178530bae5d5b

**Review by wwwy3y3 — APPROVED** [review]
LGTM

**Pull request merged**